### PR TITLE
Test against PHP 8

### PR DESCRIPTION
**Pull request by alcaeus**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -38,6 +41,12 @@ jobs:
         - wget https://scrutinizer-ci.com/ocular.phar
         - php ocular.phar code-coverage:upload --format=php-clover clover.xml
 
+    - stage: Test
+      php: nightly
+      before_script:
+        - mv ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini{.disabled,}
```
Comment (greg0ire) [inline .travis.yml:47]: This is already done in `before_install` (in a weird way, BTW, the documented way is `- phpenv config-rm xdebug.ini`), see https://docs.travis-ci.com/user/languages/php/

File: .travis.yml [hunk 2]
```diff
@@ -38,6 +41,12 @@ jobs:
...
+        - composer config platform.php 7.4.99
```
Comment (greg0ire) [inline .travis.yml:48]: This is too late, `composer update` already happened in `before_install`

File: .travis.yml (hunk 1 above)
Comment (alcaeus) [reply .travis.yml]: Ah, you're right. This should actually override `before_install`, I just used the wrong key. I've copied the original line and would suggest changing this in a future release.

Comment (greg0ire) [reply .travis.yml]: Yes let's not confuse people.

**Pull request merged**